python/lsst/meas/algorithms/gaussianPsfFactory.py

```diff
@@ -23,7 +23,7 @@
 import math
 
 from lsst.pex.config import Config, Field, ConfigurableField
-from .algorithmsLib import SingleGaussianPsf, DoubleGaussianPsf
+from . import SingleGaussianPsf, DoubleGaussianPsf
```

Comment:
Sorry, I started using shorthand on this point at some point, but that's not helpful to someone who didn't read my comments in the same order I made them.

When you import a non-module symbol directly from the current package, you're assuming the `__init__.py` for the package imports the module that defines those symbols and lifts them all to the package namespace *before* the current module is imported by that `__init__.py` file.  The alternative is
```
from .singleGaussianPsf import SingleGaussianPsf
from .doubleGaussianPsf import DoubleGaussianPsf
``
So while doing it @pschella's way protects against this module having to know what subpackage/module defines the symbols it wants, my way does not depend on the order modules are imported in `__init__.py`.  After seeing how common this is, I'm no longer completely convinced mine is better (as per my JIRA comment).